### DISCUSSION TITLE
What does the naming of the released binaries mean?

### DISCUSSION THREAD

**Discussion by magical-heyrovsky** [jarun/nnn#1787] [Q&A] [answered]
https://github.com/jarun/nnn/releases/tag/v4.9

What is the difference between the following binaries? Which one should I use in which situation? Is the last one the "default" nnn?

- nnn-emoji-static-4.9.x86_64.tar.gz
- nnn-icons-static-4.9.x86_64.tar.gz
- nnn-musl-static-4.9.x86_64.tar.gz
- nnn-nerd-static-4.9.x86_64.tar.gz
- nnn-static-4.9.x86_64.tar.gz

**Comment by N-R-K** [comment, marked as answer]
nnn [supports 3 type of icons](https://github.com/jarun/nnn/wiki/Advanced-use-cases#file-icons). The names refers to which icon pack has been enabled.

And the `musl` variant refers to the binary that's been statically linked with musl libc instead of glibc (if you don't know what musl/glibc are then you don't need to worry about it). 

> Is the last one the "default" nnn?

You can consider it that. It has icon support disabled.
